### DISCUSSION TITLE
404 not found when using Jetstream+Inertia

### DISCUSSION THREAD

**Discussion by Madhukarsprabhakara** [inertiajs/inertia#705] [Help]
Hey Folks,

So I am building this app using Jetstream+Inertia and I am running into problem when defining a route that is bound to a model:

For example, in my web.php file I have defined the following route

```
Route::get('/problems/{problem}',[App\Http\Controllers\ProblemController::class, 'show'])->name('problems.show');
```

In my vue file I have this,

```
<inertia-link :href="route('problems.show',{ problem: '1' })">
	<h1 id="message-heading" class="text-lg font-bold text-gray-900">
          Not having flexible data reports diminishes value of IC 10x
	</h1>
</inertia-link>
```

When I click on the link which actually translates to /problems/1 - it gives me 404 error

Any clues why that might be the case? Many thanks for your help.

**Comment by Maninder-cell** [comment]
chances may be something wrong with your controller or 1 problem would not present in your database
